src/main/java/net/sf/jabref/model/entry/BibEntry.java

```diff
@@ -75,11 +76,7 @@ public BibEntry() {
     }
 
     public BibEntry(String id) {
-        this(id, EntryTypes.getTypeOrDefault("misc"));
-    }
-
-    public BibEntry(String id, EntryType type) {
-        this(id, type.getName());
+        this(id, "misc");
```

Comment:
Maybe all these misc strings should be collected in a static field "DefaultType"